lib/utils/extensions/payment_title_extension.dart

```diff
@@ -0,0 +1,5 @@
+extension PaymentTitleExtension on String {
+  bool get isDefaultTitleWithLiquidNaming {
```

Comment:
I agree the implementation is a bit flaky.

The default descriptions were added to identify payment types afaik as a product decision, which could be used as payment titles on UIs but since we're not interested in using Liquid naming on Misty Breez, there's no easy way to opt-out of this. This was before `PaymentDetails` enum was added.

If other Liqud SDK users are not interested in these default descriptions, we can remove them or move them to another optional field so it's easy to opt-out of them.

Reply:
I think we don't need a default description at the sdk level as we have payment details that shows exactly what is the payment type and which direction. Users can format any default description they want in these cases. @kingonly